sapi/fuzzer/fuzzer-tracing-jit.c

```diff
@@ -69,9 +69,9 @@ int LLVMFuzzerInitialize(int *argc, char ***argv) {
 		"opcache.jit_hot_side_exit=1\n"
 		"opcache.jit_max_root_traces=32768",
 		opcache_path);
-	free(opcache_path);
```

Comment:
Any reason why this is delayed?

Reply:
to wait ini_buf is fuzzed first.

Reply:
The content of `opcache_path` is copied over, so it's no longer used. IMO it's better to deallocate things early. But I'm certainly not hard-set on this.